pkg/auth/jwt.go

```diff
@@ -0,0 +1,64 @@
+package auth
+
+import (
+	"fmt"
+	"net/http"
+
+	"NKSS-backend/pkg/handlers"
+
+	"github.com/golang-jwt/jwt"
+	_ "github.com/lib/pq"
```

Comment:
why do we need postgres here?

Reply:
That was most likely a mistake while I tried to wrestle with package not being found errors, or just me copy pasting from another file in the project to get started. I'll remove that too since it's clearly not required.